README.md

```diff
@@ -19,3 +19,7 @@ ICU4X, or "ICU for X", will be built from the start with several key design cons
 4. Written by i18n experts to encourage best practices.
 
 ICU4X will provide an ECMA-402-compatible API surface in the target client-side platforms, including the web platform, iOS, Android, WearOS, WatchOS, Flutter, and Fuchsia, supported in programming languages including Rust, JavaScript, Objective-C, Java, Dart, and C++.
+
+## Documentation
+
+The effort documentation can be found in [the docs subdirectory](docs/index.md).
```

Comment:
More information about the project can be found in [the docs subdirectory](docs/index.md).

Reply:
Done